Chiazokam/Politico
# #163504416 create the create office endpoint

#### What does this PR do?
This PR tests and creates the Create office endpoint

#### Description of Task to be completed?
- write tests for every test case
- write validators for the input
- modify the admin check for the create party endpointn
- create the create office endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-create-office-163504416`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/offices `
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
	"officeName": "President",
      "officeType": "Federal"
}
`
#### What are the relevant pivotal tracker stories?
[#163504416](https://www.pivotaltracker.com/story/show/163504416)

### Images
![image](https://user-images.githubusercontent.com/26940294/51872166-cde5e280-2358-11e9-83b8-d087f97c8f8e.png)
